apps/payments/wise.py

```diff
@@ -249,7 +228,7 @@ def _collect_bank_accounts(borderless_account):
 
         if account.bankDetails.currency == "GBP":
             # bankCode is the SWIFT code for non-GBP accounts.
-            sort_code = account.bankDetails.bankCode
+            sort_code = account.bankDetails.bankCode.replace("-", "")
```

Comment:
Similar to the handling of `accountNumber`, it might be nice for `pywisetransfer` to try to hide some of the formatting details here (depending on the class of institution codes that `bankCode` is intended to represent; I'm not sure off the top of my head what that is)

Reply:
That would be nice for the `accountNumber` issue (which I haven't encountered yet, I wonder if we can get in touch with someone about that). Returning sort codes without hyphens might be surprising for users, so we'd probably still need this code even if pywisetransfer did some normalisation.